docs/TermsAndConditionsForTheScrum.md

```diff
@@ -79,6 +79,19 @@ Applying any "priority/" label is only ok before the backlog refinement session,
 important to the team, or important to support an upcoming key result. During the refinement session, the issue should
 be planned accordingly.
 
+### Zero-bug policy
+
+In order to simplify planning, we implement a zero-bug policy. This means that
+we prioritize bugs above all other issues, **for a certain definition of bug**.
+During triage, SIGs should label sufficiently problematic issues (how they
+determine that is up to the SIGs) with `kind/bug`. These issues are prioritized
+for the next sprint.
+
+Other "bugs" (somethings does not work as it should) should be requalified. #TODO: as what
```

Comment:
I don't think that re-qualifying is necessarily a good idea. This is basically a repackaging of bug triaging which is what the zero bug policy is trying to get rid of. If we want to create a zero bug policy, I think that a bug should be narrowly defined.

Reply:
I tend to agree but reporters won't necessary have the same definition. So something reported as a bug but falling outside of that definition needs to be evaluated, and eventually re qualified if we consider it rather an improvement/feature.